test/functional/tools/explicit_conversion.xml

```diff
@@ -0,0 +1,25 @@
+<tool id="explicit_conversion" name="explicit_conversion">
+  <command>
+    cut -f 1 '${input1_table}' > 'col_output'
+  </command>
+  <inputs>
+    <param type="data" format="fasta" name="input1" label="Input 1">
+      <conversion name="input1_table" type="tabular" />
+    </param>
+  </inputs>
+  <outputs>
+    <data name="output1" format="tabular" from_work_dir="col_output" />
+  </outputs>
+  <tests>
+    <!-- Test implicit conversion. -->
+    <test>
+      <param name="input1" value="1.fasta" ftype="fasta" />
+      <param name="col" value="1" />
```

Comment:
Indeed - rebased with this change and also remove the line in the file this was copied and pasted from.